x-pack/plugins/siem/public/components/with_hover_actions/index.tsx

```diff
@@ -6,8 +6,17 @@
 
 import { EuiPopover } from '@elastic/eui';
 import React, { useCallback, useMemo, useState } from 'react';
+import styled from 'styled-components';
+
 import { IS_DRAGGING_CLASS_NAME } from '../drag_and_drop/helpers';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const WithHoverActionsPopover = (styled(EuiPopover as any)`
```

Comment:
Nit: I think you can remove both the `as any` and the corresponding eslint directive as it's redundant with the `as unknown`.

Reply:
Nope, I'm wrong, `styled-components` won't accept `EuiPopover` as an argument, the `as any` is also necessary.